### PR TITLE
Add README file for Indonesian

### PR DESCRIPTION
**Pull request by Anjianto**
## What kind of change does this PR introduce?

Add README file for Indonesian

## What is the current behavior?

- 

## What is the new behavior?

- Screenshot from 2021-05-02 20-04-22

## Additional context

-

### CODE REVIEW

**Review by GetItBac92 — CHANGES_REQUESTED** [review]
Who approved these changes

**Comment by AngSanley** [comment]
I think the first sentence "Supabase adalah sumber terbuka alternatif Firebase" is a bit wrong. It sounded like a computer-translated sentence. 

It should be "Supabase adalah alternatif sumber terbuka dari Firebase"

Also, "kita" (reader included) in the second sentence should be "kami" (reader excluded).

**Comment by Anjianto** [comment]
> I think the first sentence "Supabase adalah sumber terbuka alternatif Firebase" is a bit wrong. It sounded like a computer-translated sentence.
> 
> It should be "Supabase adalah alternatif sumber terbuka dari Firebase"
> 
> Also, "kita" (reader included) in the second sentence should be "kami" (reader excluded).

"Supabase adalah sumber terbuka alternatif Firebase" this is purely from me, not from computer-translated.

'Also, "kita" (reader included) in the second sentence should be "kami" (reader excluded).' Oh yeah, sorry i'm forgot about it.

**Comment by kiwicopple** [comment]
Thanks everyone - also note that we split the "Languages" section out to a separate file: 
https://github.com/supabase/supabase/blob/master/i18n/languages.md

File: i18n/README.id.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+## Dokumentasi
+
+Untuk dokumentasi lengkap, kunjungi [supabase.io/docs](https://supabase.io/docs)
+
+## Komunitas & Bantuan
+
+- [Forum Komunitas](https://github.com/supabase/supabase/discussions). Terbaik utnuk: membantu dengan mengembangkan, diskusi tentang praktik terbaik database.
+- [GitHub Issues](https://github.com/supabase/supabase/issues). Terbaik untuk: bug dan kesalahan yang kamu temui saat menggunakan Supabase.
+- [Email Support](https://supabase.io/docs/support#business-support). Terbaik untuk: masalah - masalah dengan database atau infrastrukturkamu
```
Comment (dnswd) [inline i18n/README.id.md:27]: There should be a space between `infrastruktur` and `kamu`.

File: i18n/README.id.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+# Supabase
+
+[Supabase](https://supabase.io) adalah sumber terbuka alternatif Firebase. Kita mengembangkan fitur Firebase menggunakan alat sumber terbuka kelas perusahaan.
+
+- [x] Menggunakan database Postgres yang di host
+- [x] Langganan waktu sebenarnya
+- [x] Otentikasi dan otorisasi
+- [x] API yang dibuat secara otomatis
+- [x] Dashboard
```
Comment (dnswd) [inline i18n/README.id.md:15]: You can use `Dasbor` to replace `Dashboard`. But you can leave it as is, it's fine already.

File: i18n/README.id.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+## Komunitas & Bantuan
+
+- [Forum Komunitas](https://github.com/supabase/supabase/discussions). Terbaik utnuk: membantu dengan mengembangkan, diskusi tentang praktik terbaik database.
+- [GitHub Issues](https://github.com/supabase/supabase/issues). Terbaik untuk: bug dan kesalahan yang kamu temui saat menggunakan Supabase.
+- [Email Support](https://supabase.io/docs/support#business-support). Terbaik untuk: masalah - masalah dengan database atau infrastrukturkamu
+
+## Status
+
+- [x] Alpha: Kita test Supabase dengan sejumlah pelanggan tertutup.
```
Comment (dnswd) [inline i18n/README.id.md:31]: This sounds too stiff, you can improve it by replacing `test` with `menguji`. Also I think it's better to add `secara` between `pelanggan` and `tertutup`.

`Kami menguji Supabase dengan sejumlah pelanggan secara tertutup` sounds a bit better.

File: i18n/README.id.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+Kita sekarang sedang di Public Beta. Watch "rilis" dari repo ini untuk mendapatkan pembaruan besar.
+
+<kbd><img src="https://gitcdn.link/repo/supabase/supabase/master/web/static/watch-repo.gif" alt="Watch this repo"/></kbd>
+
+---
+
+## Bagaimana ini berkerja
+
+Supabase adalah kombinasi dari alat - alat sumber terbuka. Kita mengembangkan fitur Firebase menggunakan produk sumber terbuka kelas perusahaan. Jika alat - alat dan komunias - komunitas ada, dengan MIT, Apache 2, atau yag sama dengan open license, kita akan menggunakan dan dukung alat itu. Jika alat itu tidak ada, kita buat dan kita akan membuka kode itu. Supabase tidak 1 untuk 1 pemetaan Firebase. Tujuan kami adalah untuk memberi pengembang sebuah pengalaman seperti Firebase pengembang menggunakan alat -alat sumber terbuka.
```
Comment (dnswd) [inline i18n/README.id.md:44]: - Delete spaces between hypen:
  - `alat - alat` to `alat-alat`
  - `komunitas - komunitas` to `komunitas-komunitas`


- > Supabase tidak 1 untuk 1 pemetaan Firebase.  Tujuan kami adalah untuk memberi pengembang sebuah pengalaman seperti Firebase pengembang menggunakan alat -alat sumber terbuka.

  This sounded like a computer-translated sentence. You can rephrase this.

- Several typos.

File: i18n/README.id.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+**Arsitektur saat ini**
+
+Supabase adalah sebuah [platform yang dihosting](https://app.supabase.io). Kamu bisa daftar dan mulai menggunakan Supabase tanpa menginstall apapun. Kita tetap membuat pengalaman mengembakan di lokal - ini sekarang tujuan utama kita, selama dengan platform yang stabil.
```
Comment (dnswd) [inline i18n/README.id.md:48]: - The standard translation for `install` is `pasang`. So you can replace `menginstall` with `memasang`.

- > selama dengan platform yang stabil.

  I don't think this is the correct translation.

File: i18n/README.id.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+![Arsitektur](https://supabase.io/assets/images/supabase-architecture-9050a7317e9ec7efb7807f5194122e48.png)
+
+- [PostgreSQL](https://www.postgresql.org/) adalah obejct relational database sistem dengan pengembangan aktif lebih dari 30 tahun yang membuatnya memiliki reputasi yang kuat untuk keandalan, fitur ketahanan dan kinerja.
```
Comment (dnswd) [inline i18n/README.id.md:52]: This translation is also sounds off.

**Review by dnswd — CHANGES_REQUESTED** [review]
This is a fairly nice translation, but I agree with @AngSanley. There are lots of lines that can be improved.

**Review by Nisaju — APPROVED** [review]

**Pull request closed without merging**